tox.ini

```diff
@@ -1,5 +1,10 @@
 [tox]
-envlist = linters,docs,unit-tests
+envlist = clean,linters,docs,unit-tests
+
+[testenv:clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
```

Comment:
Will this be running in the github workflows? or just when we run `tox` locally? 
Also, could you update branch with main.

Reply:
We can add it to the workflows but for now just locally